app/components/add-validations-to-form.hbs

```diff
@@ -1,14 +1,30 @@
-{{#each this.storesWithForm as |field|}}
-  <strong>{{field.name}}</strong>
-  <br />
-  <RdfForm
-    @groupClass="au-o-grid__item"
-    @form={{field.form}}
-    @graphs={{this.graphs}}
-    @formStore={{field.store}}
-    @forceShowErrors={{false}}
-    @useNewListingLayout={{true}}
-    @sourceNode={{this.graphs.sourceNode}}
-  />
-  <br />
-{{/each}}
\ No newline at end of file
+<AuBodyContainer>
+  <div class="fieldSelectionForValidations">
+    <AuDropdown @title={{"Select field"}} @skin="primary" role="menu">
+      {{#each this.storesWithForm as |field|}}
+
+        <AuButton
+          {{on "click" (fn this.setSelectedField field)}}
+          @skin="link"
+          role="menuitem"
+        >
+          {{field.name}}
+        </AuButton>
+      {{/each}}
+
+    </AuDropdown>
```

Comment:
I tested this on the Big Form (50 fields), not everything fits on the dropdown and dropdown isn't scroll-able

Reply:
I took this form as a standard form in my head https://dev.form-builder.lblod.info/formbuilder/652661D898CA4C0008000019/edit FYI